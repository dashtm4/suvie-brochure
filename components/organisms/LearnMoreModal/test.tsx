import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { LearnMoreModal } from '.'

describe('<LearnMoreModal />', () => {
  const title = 'Test title'
  const description = 'Test description'
  const videoSrc = 'video.jpg'
  const posterSrc = 'image.jpg'
  const toggleModal = jest.fn()

  it('should renders correct <LearnMoreModal />', () => {
    const wrapper = shallow(
      <LearnMoreModal
        title={title}
        description={description}
        videoSrc={videoSrc}
        posterSrc={posterSrc}
        toggleModal={toggleModal}
      />
    )

    expect(wrapper.contains(title)).toBeTruthy()
    expect(wrapper.contains(description)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
