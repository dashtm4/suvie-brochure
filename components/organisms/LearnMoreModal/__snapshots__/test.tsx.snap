// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LearnMoreModal /> should renders correct <LearnMoreModal /> 1`] = `
<VideoModal
  posterSrc="image.jpg"
  toggleModal={[MockFunction]}
  videoSrc="video.jpg"
>
  <h3
    className="mb-4 font-serif font-normal text-center text-secondary md:text-left"
    style={
      Object {
        "fontSize": "2rem",
      }
    }
  >
    Test title
  </h3>
  <div
    className="mb-4 text-center text-quinary md:text-left"
  >
    Test description
  </div>
  <Button
    className="hidden mx-auto max-w-full md:inline-block w-50 md:mx-0"
    onClick={[MockFunction]}
    theme="outline"
  >
    Close window
  </Button>
</VideoModal>
`;
